R/tune_args.R

```diff
@@ -0,0 +1,28 @@
+#' Determine arguments tagged for tuning
+#'
+#' `tune_args()` takes an object such as a model specification or a recipe and
+#' returns a tibble of information on all possible tunable arguments and
+#' whether or not they are actually tunable.
+#'
+#' @param object A `model_spec`, `recipe`, `workflow` or other object.
+#' @param ... Other arguments passed to methods.
+#'
+#' @details
+#' The `source` column is determined differently depending on whether a
+#' `model_spec` or a `recipe` is used (with additional detail on the type).
+#'
+#' The `id` field has any identifier that was passed to [tune::tune()] (e.g.
```

Comment:
```suggestion
#' The `id` field has any identifier that was passed from [tune::tune()] (e.g.
```
??? 

This is confusing otherwise, right?